src/argilla/client/feedback/dataset.py

```diff
@@ -472,7 +472,12 @@ def iter(self, batch_size: Optional[int] = FETCHING_BATCH_SIZE) -> Iterator[List
         for i in range(0, len(self.records), batch_size):
             yield self.records[i : i + batch_size]
 
-    def push_to_argilla(self, name: Optional[str] = None, workspace: Optional[Union[str, rg.Workspace]] = None) -> None:
+    def push_to_argilla(
+        self,
+        name: Optional[str] = None,
+        workspace: Optional[Union[str, rg.Workspace]] = None,
+        show_progress: bool = False,
```

Comment:
Do you mind also including the `show_progress` arg in the docstring under the `Args` section? Thanks

Reply:
Sure would love to update the docs.